admission/Makefile

```
@@ -45,7 +36,7 @@ test: crds simple-test setup-envtest
 	}
 
 .PHONY: simple-test
-simple-test: $(STATICCHECK)
+simple-test: staticcheck
 	test -z "$$(gofmt -s -l . | tee /dev/stderr)"
 	$(STATICCHECK) ./...
```

It should be replaced together?
```suggestion
	staticcheck ./...
```

---

No.

The dependency `staticcheck` is a PHONY target.
https://github.com/cybozu/neco-containers/pull/1214/files#diff-df732126041c7620e3b28705a84144e9aa2f82e739b25f8f1d9f5e452be20fbfR65

On the other hand, the variable refers to the binary.

---

Sorry I missed the fowllowing line and was wondering about how it resolves to a real value.
```
STATICCHECK = $(PROJECT_DIR)/bin/staticcheck
```